### PR TITLE
Fix/keeper spree deployment

### PR DESCRIPTION
**Pull request by H34D**
## Description

Version 0.6 of the keeper requires to have multiple accounts to do contract deployments.

## Is this PR related with an open issue?

Related to Issue #103 https://github.com/oceanprotocol/keeper-contracts/issues/275

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,14 +147,32 @@ Hostname                    | External Ports   | Internal Url
...
 You will have available a keeper node in the local and private `spree` network with the following accounts enabled:
 
-Account                                      | Password   | Balance
----------------------------------------------|------------|--------
-`0x00bd138abd70e2f00903268f3db08f2d25677c9e` | node0      | 10000000111000111000111000
-`0x068ed00cf0441e4829d9784fcbe7b9e26d4bd8d0` | secret     | 100000000
-`0xa99d43d86a0758d5632313b8fa3972b6088a21bb` | secret     | 100000000
+Account                                      | Type     | Password/Key                 | Balance
+---------------------------------------------|----------|------------------------------|-----------------
+`0x0011598De1016A350ad719D23586273804076774` | key      |                              | 1000000000 Ether
```
Comment (H34D) [inline README.md]: @jcortejoso do you know the password? or shall we completely remove this account? From chain config and the key?

File: README.md [hunk 2]
```diff
@@ -147,14 +147,32 @@ Hostname                    | External Ports   | Internal Url
...
+`0x00Bd138aBD70e2F00903268F3Db08f2D25677C9e` | key      | node0                        | 1000000000 Ether
+`0x068Ed00cF0441e4829D9784fCBe7b9e26D4BD8d0` | key      | secret                       | 1000000000 Ether
+`0x6B0c56d1Ad5144b4d37fa6e27DC9afd5C2435c3B` | key      |                              | 1000000000 Ether
```
Comment (H34D) [inline README.md]: @jcortejoso do you know the password? or shall we completely remove this account? From chain config and the key?

File: README.md (hunk 1 above)
Comment (jcortejoso) [reply README.md]: For this account I do not have neither the json file nor the password. I think we can remove. It was in the original ethereum-playground project.

File: README.md (hunk 2 above)
Comment (jcortejoso) [reply README.md]: `0x6B0c56d1Ad5144b4d37fa6e27DC9afd5C2435c3B` has an empty password. Also we can think about removing. It is also coming from the original ethereum-playground project.

Comment (H34D) [reply README.md]: i removed it

File: README.md (hunk 1 above)
Comment (H34D) [reply README.md]: i removed it

**Pull request merged**